<ui version="4.0" >
 <class>ScanResults</class>
 <widget class="QDialog" name="ScanResults" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scan results</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="scanResultsWidget" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>5</number>
     </property>
     <column>
      <property name="text" >
       <string>SSID</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>BSSID</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>frequency</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>signal</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>flags</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="scanButton" >
       <property name="text" >
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
